src/seedu/addressbook/data/person/Person.java

```diff
@@ -83,5 +84,16 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
+    
+    public String getPrintableString(Printable... printables) {
+    	String info = new String();
+    	info = "";
+    	for(Printable print: Printable) {
+    		info.concat(print.getPrintableString());
+    		info += ", ";
```

Comment:
- This would result in your final string containing an extra comma and space at the end.
- While this implementation is fine and you would most probably not deal with a large number of `Printable` objects, this is not the most efficient way of concatenating strings in Java. If you wish to find out more about how to efficiently concatenate strings in Java, I highly recommend googling around to find answers. It's quite interesting to note how something this seemingly trivial can result in large performance costs in certain cases.